site2/docs/reference-cli-tools.md

```diff
@@ -315,6 +315,10 @@ Options
 |`-m`, `--messages`|Comma-separated string of messages to send; either -m or -f must be specified|[]|
 |`-n`, `--num-produce`|The number of times to send the message(s); the count of messages/files * num-produce should be below 1000|1|
 |`-r`, `--rate`|Rate (in messages per second) at which to produce; a value 0 means to produce messages as fast as possible|0.0|
+|`-c`, `--chunking`|Should split the message and publish in chunks if message size is larger than allowed max size|false|
+|`-s`, `--separator`|Character to split messages string on default is comma|","|
+|`-k`, `--key`|Message key to add|key=value string, like k1=v1,k2=v2.|
+|`-p`, `--properties`|Properties to add, Comma separated| |
```

Comment:
Do you mean "Properties to add. If you want to add multiple properties, use the comma as the separator."?